src/app/components/Uploader/index.tsx

```diff
@@ -0,0 +1,118 @@
+/** @jsxRuntime classic */
+/** @jsx jsx */
+import { useContext } from 'react';
+import path from 'ramda/src/path';
+import pathOr from 'ramda/src/pathOr';
+import { jsx } from '@emotion/react';
+import { ServiceContext } from '../../contexts/ServiceContext';
+import Heading from '../Heading';
+import Paragraph from '../Paragraph';
+import Text from '../Text';
+import { LeftChevron, RightChevron } from '../icons';
+import styles from './index.styles';
+
+interface UploaderProps {
+  blocks: object[] | undefined;
+}
+
+type UploaderBlock = {
+  type: 'title' | 'text' | 'link';
+  model: {
+    blocks: object[];
+  };
+};
+
+const Uploader = ({ blocks }: UploaderProps) => {
+  const { dir } = useContext(ServiceContext);
+  const isRtl = dir === 'rtl';
+
+  if (blocks === undefined) return null;
+
+  const type = path([0, 'type'], blocks); // aresUploader
+  if (type !== 'aresUploader') return null;
+
+  const uploaderBlocks = pathOr<UploaderBlock[]>(
+    [],
+    [0, 'model', 'blocks'],
+    blocks,
+  );
+
+  const titleBlock = uploaderBlocks.find(block => block.type === 'title');
+  const textBlock = uploaderBlocks.find(block => block.type === 'text');
+  const linkBlock = uploaderBlocks.find(block => block.type === 'link');
+
+  const title = pathOr<string>(
+    '',
+    ['model', 'blocks', 0, 'model', 'blocks', 0, 'model', 'text'],
+    titleBlock,
+  );
+
+  const text = pathOr<string>(
+    '',
+    ['model', 'blocks', 0, 'model', 'blocks', 0, 'model', 'text'],
+    textBlock,
+  );
+
+  const linkText = pathOr<string>(
+    '',
+    [
+      'model',
+      'blocks',
+      0,
+      'model',
+      'blocks',
+      0,
+      'model',
+      'blocks',
+      0,
+      'model',
+      'text',
+    ],
+    linkBlock,
+  );
+
+  const linkAddress = pathOr<string>(
+    '',
+    [
+      'model',
+      'blocks',
+      0,
+      'model',
+      'blocks',
+      0,
+      'model',
+      'blocks',
+      0,
+      'model',
+      'locator',
+    ],
+    linkBlock,
+  );
+
+  const id = `${title.replaceAll(' ', '-')}`;
+
+  return (
+    <section role="region" aria-labelledby={id} css={styles.container}>
+      <div css={styles.card}>
+        <Heading level={2} size="paragon" css={styles.heading} id={id}>
+          {title}
+        </Heading>
+        <Paragraph>{text}</Paragraph>
+        <div css={styles.linkContainer}>
+          <a href={linkAddress} css={styles.linkBackground}>
+            <Text size="pica" fontVariant="sansBold" css={styles.link}>
+              {linkText}
+              {isRtl ? (
+                <LeftChevron css={styles.chevron} />
+              ) : (
+                <RightChevron css={styles.chevron} />
+              )}
+            </Text>
+          </a>
+        </div>
+      </div>
+    </section>
```

Comment:
I hope you don't mind me snooping on this PR 🕵️‍♀️ - this looks very similar to the CTA Link we have within the Message Banner (see https://github.com/bbc/simorgh/blob/d9b325156fd69aa79c92e9c0cafd51f86e2bf0a9/src/app/components/MessageBanner/index.tsx#L40-L65), and I wondered if now is the perfect opportunity for a refactor - have a new CTALink component within src/app/components, and use it both here and in the Article Page?

Reply:
Ah yes that's a good idea. Especially since I looked (quite closely 👀) to the MessageBanner for reference! I will update.

Reply:
Update: Working on this here - https://github.com/bbc/simorgh/pull/10783

Reply:
Update: now merged